src/inversion_scripts/operators/TROPOMI_operator.py

```diff
@@ -323,6 +323,11 @@ def apply_tropomi_operator(
                 coords[1] + dlat / 2,
                 coords[1] + dlat / 2,
             ]
+            # If this is a global 2.0 x 2.5 grid, extend the eastern-most grid cells to 180 degrees
+            if (dlon == 2.5) & (coords[0] == 177.5):
+                for i in [1,2]:
+                    geoschem_corners_lon[i] += dlon / 2
+
```

Comment:
@megan-he @eastjames this is not a big deal, just flagging it because it's kind of weird. To ensure assert abs(sum(overlap_area)-polygon_tropomi.area)/polygon_tropomi.area < 0.01 is true (that is, that we find a GEOS-Chem grid cell for all TROPOMI pixels which should be true on a global grid), I had to expand the eastern-most grid boxes on the 2.0 x 2.5 grid to 180 degrees (because the centers go from -180 to 177.5, so 178.75-180 did not exist!) Might be relevant elsewhere, but not a big deal.

Reply:
Thanks Nick! I had to change the max longitude of my state vector from 180 to 177.5 previously to avoid index mismatching, so I think this should resolve that issue.

Reply:
Looks good @nicholasbalasus!